lib/rules/web/admin_console/4.15/common_ui_elements.xyaml

```diff
@@ -475,7 +475,7 @@ check_resources_title_on_page:
 click_kebab_item:
   element:
     selector:
-      xpath: //button[contains(@class,'pf-c-dropdown__menu-item') and not(contains(@class,'pf-m-disabled')) and text()='<kebab_item>']
+      xpath: //button[contains(@data-test-action,'<kebab_item>') and not(contains(@class,'pf-m-disabled'))]
```

Comment:
@XiyunZhao this part should not need now:  and not(contains(@class,'pf-m-disabled')

Reply:
@yanpzhan Okay, sure, I will remove the 'not' part